compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/DerivedConfigUtils.cpp

```diff
@@ -181,4 +181,31 @@ SmallVector<int64_t> deriveThreadTileSizes(Operation *op) {
       .Default([&](Operation *op) -> SmallVector<int64_t> { return {}; });
 }
 
+static const int64_t kDefaultGlobalLoadBitSizePerThread =
+    32; // TODO: make it a query.
```

Comment:
nit: move comment to separate line to make the code formatter happier.
```suggestion
 // TODO: make it a query.
static const int64_t kDefaultGlobalLoadBitSizePerThread = 32;
```